rasa-ext-vim/src/Rasa/Ext/Vim.hs

```diff
@@ -83,10 +83,10 @@ anyMode _ = return ()
 
 -- | Listeners for keypresses when in 'Insert' mode
 insert :: [Keypress] -> BufAction ()
-insert [KEsc] = mode .= Normal
+insert [KEsc _] = mode .= Normal
 
-insert [KBS] = moveRangesByN (-1) >> delete
-insert [KEnter] = insertText "\n" >> moveRangesByC (Coord 1 0) >> startOfLine
+insert [KBS _] = moveRangesByN (-1) >> delete
+insert [KEnter _] = insertText "\n" >> moveRangesByC (Coord 1 0) >> startOfLine
```

Comment:
I'd prefer that we be specific about what we want for these; we probably only want this to act like Enter when there's no mods; same for the others, (probably for insertText too). Can we change these to `KEnter []`  and friends? Thanks!